content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -61,12 +58,12 @@ feature, not all CRIs support multiple RuntimeClasses yet.
 {{< note >}}
 **Note:** RuntimeClass currently assumes a homogeneous node configuration across the cluster
 (which means that all nodes are configured the same way with respect to container runtimes). Any heterogeneity (varying configurations) must be
-managed independently of RuntimeClass through scheduling features (see [Assigning Pods to
-Nodes](/docs/concepts/configuration/assign-pod-node/)).
+managed independently of RuntimeClass through scheduling features
+(see [Assigning Pods to Nodes](/docs/concepts/configuration/assign-pod-node/)).
 {{< /note >}}
 
 The configurations have a corresponding `RuntimeHandler` name, referenced by the RuntimeClass. The
-RuntimeHandler must be a valid DNS 1123 subdomain (alpha-numeric + `-` and `.` characters).
+RuntimeHandler must be a valid DNS 1123 subdomain (alpha-numeric, `+`, `-`, and `.` characters).
 
```

Comment:
Change seems wrong, `a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.'`, so plz replace `+` with `,`.

Reply:
oh, yeah, you are right. Can I update the line to be like:
```
valid DNS-1123 subdomain (alpha-numeric characters, `-` or `.`)
```